Sources/PointFree/__Generated__/DerivedPartialIsos.swift

```diff
@@ -139,6 +139,18 @@ import Prelude
       }
 
 
+      extension PartialIso where A == (
+        String
+      ), B == Route {
+
+        public static let discounts = parenthesize <| PartialIso(
+          apply: Route.discounts,
```

Comment:
sourcery is completely butchering this file now so I'm just copying and pasting these partial iso helpers.

Reply:
:(